src/inference/tests/unit/core.cpp

```diff
@@ -381,6 +381,11 @@ TEST(CoreTests_parse_device_config, get_device_config) {
         "HETERO",
         ov::AnyMap{ov::device::priorities("MULTI,DEVICE"),
                    ov::device::properties(ov::AnyMap{{"MULTI", ov::AnyMap{ov::device::priorities("DEVICE")}}})});
+
+    // invalid device name with characters after parenthesis except comma
+    EXPECT_THROW(ov::parseDeviceNameIntoConfig("DEVICE(0)ov", ov::AnyMap{}), ov::Exception);
+    EXPECT_THROW(ov::parseDeviceNameIntoConfig("MULTI:DEVICE(0)ov,DEVICE(1)", ov::AnyMap{}), ov::Exception);
```

Comment:
If comma is located in the last character, the device name should be invalid, such as "MULTI:DEVICE(0),DEVICE(1),"

Reply:
"MULTI:DEVICE(0),DEVICE(1)," is valid as a device name and I add this test case to the unit tests.

Reply:
`"MULTI:DEVICE(0),DEVICE(1),"` is invalid device name, isn't it?

Reply:
@riverlijunjie Yes, it is a invalid name and I fix this case, please help me review the PR. Thanks!